tests/parsing/infiniteLoops/nonRecTypes.res

```diff
@@ -20,48 +20,48 @@ include {
             ) =>
             t('value) =
             ""
-            "BS:6.0.1\132\149\166\190\000\000\000#\000\000\000\r\000\000\000&\000\000\000#\145\160\160A\160$size@\160\160A\160$root@\160\160A\160'compare@@";
+            "BS:6.0.1\x84\x95\xa6\xbe\0\0\0#\0\0\0\r\0\0\0&\0\0\0#\x91\xa0\xa0A\xa0$size@\xa0\xa0A\xa0$root@\xa0\xa0A\xa0'compare@@";
```

Comment:
I converted tests strings using the playground and copy/pasting the string from the JS output.